easybuild/easyblocks/g/gamess_us.py

```diff
@@ -185,11 +186,12 @@ def configure_step(self):
 
         # patch hardcoded settings in rungms to use values specified in easyconfig file
         rungms = os.path.join(self.builddir, 'rungms')
-        extra_gmspath_lines = "set ERICFMT=$GMSPATH/auxdata/ericfmt.dat\nset MCPPATH=$GMSPATH/auxdata/MCP\n"
+        extra_gmspath_lines = "set ERICFMT=$GMSPATH/auxdata/ericfmt.dat\nset MCPPATH=$GMSPATH/auxdata/MCP"
+        extra_rdma_line = "set USE_RDMA"
         try:
             for line in fileinput.input(rungms, inplace=1, backup='.orig'):
```

Comment:
ideally, use of `fileinput` should be replace with using `apply_regex_substitutions`, see http://easybuild.readthedocs.org/en/latest/Extended_dry_run.html#runtime-patching-of-files-apply-regex-substitutions